Automattic/wp-calypso
# Social Logos: update to latest version and add Bluesky colors

Related to https://github.com/Automattic/jetpack/pull/36091/

## Proposed Changes

Update to the latest version of the Social Logos package (last updated as of https://github.com/Automattic/social-logos/pull/131 ), and add the Bluesky colors so the sharing button screen can include the Bluesky icon.

## Testing Instructions

* You'll need a local installation of Jetpack running the branch from https://github.com/Automattic/jetpack/pull/36091/
* Spin up Calypso locally with this branch and go to http://calypso.localhost:3000/marketing/sharing-buttons/
* Click to edit the sharing buttons
* You should see the Blue Bluesky logo

## Pre-merge Checklist
